frontend/src/utils/formatStatus.ts

```diff
@@ -0,0 +1,8 @@
+import { TripStatusStatusEnum } from "@/api_gen";
+
+export default function formatStatus(status: TripStatusStatusEnum) {
```

Comment:
Ja bi tu mozda radije napravio neki objekt koji definira labelu za svaki trip status enum. To bi izgledalo nekako ovako

```typescript
const tripStatusFormat: Record<TripStatusEnum, string> = {
   TripStatusEnum.DepartmentApprovalRejected: "Department approval rejected"
   ...
}
```

Reply:
65a70d86fc778f2ce1a26b9c32f8af310c8e829f